miner/util.go

```diff
@@ -0,0 +1,38 @@
+package miner
+
+import (
+	"fmt"
+
+	"golang.org/x/exp/maps"
+
+	"github.com/spacemeshos/go-spacemesh/common/types"
+	"github.com/spacemeshos/go-spacemesh/sql"
+	"github.com/spacemeshos/go-spacemesh/sql/ballots"
+	"github.com/spacemeshos/go-spacemesh/sql/blocks"
+	"github.com/spacemeshos/go-spacemesh/sql/certificates"
+)
+
+func ActiveSetFromBlock(db sql.Executor, epoch types.EpochID) ([]types.ATXID, error) {
```

Comment:
I think the naming doesn't fit. It generates the ActiveSet from the first block in the provided epoch. Maybe `EpochActiveSet` or similar is a better name?

Reply:
EpochActiveSet means many things. it can mean the active set is derived from
- hare active set (determined by hare rules)
- tortoise active set (all epoch atx ids)
- block active set (derived from block)
- fallback (overwritten manually)

i think the naming fits here.

Reply:
It is confusing to me because instead of passing in a Block ID, I have to pass in an Epoch ID.

Reply:
i guess i can rename it to ActiveSetFromFirstBlockInEpoch?

Reply:
Yeah, I cannot think of a better name and you already stated that `EpochActiveSet` would be confusing for other reasons.